torch/_inductor/decomposition.py

```diff
@@ -212,18 +254,15 @@ def should_pad_bench(mat1, mat2, op, input=None):
                 lambda: op(input, mat1, mat2),
             )
 
-        mat1_pad = torch.randn_like(mat1)
-        mat2_pad = torch.randn_like(mat2)
-
         if op is torch.ops.aten.addmm:
             input_pad = None
             if input is not None and input.is_cuda:
                 input_pad = torch.randn_like(input)
             pad_time = do_bench(
                 lambda: pad_addmm(
                     input_pad,
-                    mat1_pad,
-                    mat2_pad,
+                    mat1,
```

Comment:
could it be that you are reusing original inputs here, instead of making a copy (the source of your weird error)

Reply:
I got the same error with no changes here https://github.com/pytorch/pytorch/pull/101619/. `mat1` and `mat2` are reassigned here, they're not the original.